test/extended/testdata/build-secrets/s2i-binary-dir/.s2i/bin/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Ensure none of the build config inject secrets still exist in the file system
+for s in $(ls /tmp/secret? secret?); do
+    if [ -s "${s}" ]; then
+        echo "Found secret file which should have been removed: ${s}"
+        exit 1
```

Comment:
@bparees s2i appears to have the contract that it will remove secret files injected during a build by a BuildConfig prior to completing the docker image. This check ensures their absence at run. If a secrete file was not removed by the time run executes, the stdout of run will contain the "Found secret file.." message. Any deviation (missing lines or additional lines) causes the test case to fail.